test/test_model.py

```diff
@@ -207,6 +207,52 @@ def test_model_info(self):
         self.assertIn('theta', model_info_include['parameters'])
         self.assertIn('included_files', model_info_include)
 
+    def test_compile_with_includes(self):
+        getmtime = os.path.getmtime
+        configs = [
+            ('add_one_model.stan', ['include-path']),
+            ('bernoulli_include.stan', []),
+        ]
+        for stan_file, include_paths in configs:
```

Comment:
Sorry to be nit-picky, can you turn this into a pytest fixture?

e.g.
```python
    @pytest.mark.parametrize("stan_file,include_paths", [('add_one_model.stan', ['include-path']),('bernoulli_include.stan', [])])
    def test_compile_with_includes(self, stan_file, include_paths):
        ...

Reply:
> Oh, bummer. More of a reason to ditch TestCase one day.

It looks like moving the tests fully to the `pytest` framework wouldn't be too much work. Interest in a PR?

Reply:
Good question. I’m not sure why we have our tests in the unittest style, and I would be okay with starting to ditch that, but we should probably open an issue to discuss more. 

I have a few gripes with the testing we do, we have a lot of older tests which rely on internal behavior of the classes and make refactors (even backwards compatible ones) very annoying

Reply:
I played around with doing the translation while my brain was stuck on something else. I'll open a draft PR and we can discuss from there?

Reply:
Sounds good!